### ISSUE TITLE
warning compile Ubuntu 21.04

### ISSUE THREAD

**Issue by idk-whoami** [rathena/rathena#7417]
### rAthena Hash

a763ad06294f1478343b3e765064e4bcdcb993c0

### Client Date

20211103

### Server Mode

Renewal

### Result

![Capture](https://user-images.githubusercontent.com/60462533/200481834-5234850d-4509-4b4a-8561-e129bdd1d9c0.PNG)

```
        CXX     navi.cpp
        CXX     npc_chat.cpp
        CXX     npc.cpp (custom rule)
npc.cpp: In function ‘int npc_dynamicnpc_removal_timer(int, t_tick, int, intptr_t)’:
npc.cpp:5781:166: warning: passing NULL to non-pointer argument 4 of ‘int add_timer(t_tick, TimerFunc, int, intptr_t)’ [-Wconversion-null]
 cnpc.last_interaction + battle_config.feature_dynamicnpc_timeout, npc_dynamicnpc_removal_timer, nd->bl.id, NULL );
                                                                                                                 ^
npc.cpp:5787:181: warning: passing NULL to non-pointer argument 4 of ‘int add_timer(t_tick, TimerFunc, int, intptr_t)’ [-Wconversion-null]
 action + DIFF_TICK( gettick(), nd->dynamicnpc.last_interaction ), npc_dynamicnpc_removal_timer, nd->bl.id, NULL );
                                                                                                                 ^
npc.cpp: In function ‘npc_data* npc_duplicate_npc_for_player(npc_data&, map_session_data&)’:
npc.cpp:5834:167: warning: passing NULL to non-pointer argument 4 of ‘int add_timer(t_tick, TimerFunc, int, intptr_t)’ [-Wconversion-null]
 npc.last_interaction + battle_config.feature_dynamicnpc_timeout, npc_dynamicnpc_removal_timer, dnd->bl.id, NULL );
                                                                                                                 ^
        CXX     party.cpp
        CXX     path.cpp
        CXX     pc.cpp
        CXX     pc_groups.cpp
        CXX     pet.cpp
```

### Expected Result

-

### How to Reproduce

-

### Official Information

-

**Comment by mazvi** [comment]
any update for this?

**Comment by secretdataz** [comment]
Update: fix is still inside of #7410
